test/ITensorLegacyMPS/base/test_dmrg.jl

```diff
@@ -110,7 +110,10 @@ using ITensors: nsite, set_nsite!
     @test length(PHdisk) == N
     @test ITensors.site_range(PH) == n:(n + 1)
     @test eltype(PH) == Float64
-    @test size(PH) == (3^2 * 4^2, 3^2 * 4^2)
+    ## TODO sometimes randomMPS gives a linkdim value of 3 
+    ## which causes an error in `size = 3^2 * 4^2`
+    size = linkdim(psi, n)^2 * dim(siteind(psi, n))^2
+    @test size(PH) == (size, size)
```

Comment:
This isn't quite right. `PH` is the MPO projected onto sites `n` and `n + 1`, so it should be:
```julia
size = linkdim(psi, n - 1) * linkdim(psi, n + 1) * dim(siteind(psi, n)) * dim(siteind(psi, n + 1))
```

Reply:
Ahh I see, I updated the code. Thanks!!